src/main/java/Task.java

```diff
@@ -0,0 +1,57 @@
+import java.time.LocalDateTime;
+
+public class Task{
+    private String content;        // stores the name of this task
+    private Boolean closed = false;     //indicates the status of this task

```

Comment:
Perhaps consider using isClosed instead of Closed? This will make it sound more like a boolean.